opm/output/eclipse/EclipseIO.hpp

```diff
@@ -107,7 +107,7 @@ class EclipseIO {
   *     are not yet written to disk.
   */
 
-    void writeInitial( data::Solution simProps = data::Solution(), const NNC& nnc = NNC());
+    void writeInitial( data::Solution simProps = data::Solution(), std::map<std::string, std::vector<int> > map = {}, const NNC& nnc = NNC());
```

Comment:
Please add a brief documentation of the new map for integer data, in particular the requirements on the length of the vector:

1. The on-disk keywords will always have `nx*ny*nz` elements.
2. Vectors of length `nactive` will be expanded to length `nx*ny*nz` - inactive cells witll be filled with -1.
3. Others lengths -> exception.

 should be documented - the current choice is not obvious, and people might expect/wish different behavior.